documentation/book/cluster-operator.adoc

```diff
@@ -521,13 +524,14 @@ memory used by the JVM.
 [[kafka_rack]]
 ===== Kafka rack
 
-It is possible to enable Kafka rack-awareness by specifying a JSON string for the `kafka-rack` field in the cluster ConfigMap.
-Such a JSON object has one mandatory field named `topologyKey`.
+It is possible to enable Kafka rack-awareness (more information can be found on the https://kafka.apache.org/documentation/#basic_ops_racks[Kafka website])
```

Comment:
Can you change the Kafka link (URL and name) to a variable in `attributes.adoc`? We might need to brand it downstream and replace it with our own documentation.